### DISCUSSION TITLE
Can i get banned for embeding Eaglecraft (minecraft clone)?

### DISCUSSION THREAD

**Discussion by 11NightFox11** [community/community#54051] [Other Feature Feedback, Questions, & Ideas]
Question

Can i get banned for embeding game named Eaglecraft (minecraft remake)?
Offical webiste was closed becouse mojang blocked it.
For the moment you cant play this game but i have working link to it

So can i iframe this game without getting banner from github?
(I also want to publish this website)

Sorry for bad English

**Comment by ThatManTrapBoi** [comment]
idk rlly (I'm 3 years late lol)

**Comment by ushxyuki** [comment]
Hi!

I would not recommend embedding it, especially if the official website was closed because of Mojang/Microsoft action.

GitHub Pages should not be used to host or link to content that may violate copyright, trademarks, or someone else’s terms of service. Even if you only use an iframe, you could still risk your repository/site being reported or removed.

The safest option is to avoid embedding it unless you have clear permission from the game owner or the project is legally allowed to be shared.

Hope this helps!

**Comment by AlexCojescu** [comment]
Yeah, you'll get your account banned. Don't do this. Embedding a link to a blocked game in an iframe doesn't protect you. GitHub's Terms of Service explicitly prohibit facilitating access to copyrighted material, and that's exactly what you'd be doing. An iframe that loads a working game is still distributing the game the distinction between "hosting files" and "linking to files" doesn't matter legally.

GitHub Pages runs automated scanners that flag deployed sites linking to known pirated content. Microsoft also has bots that actively crawl repositories looking for Minecraft clones and alternative hosting URLs. The moment your repo goes public, their legal team will file a DMCA takedown. GitHub won't warn you, they'll just delete the repository.

If it's your first offense on an established account, they'll disable that specific repository and turn off your GitHub Pages deployment. You keep your account, but with a strike on your record. If you're linking to multiple mirrors or creating throwaway accounts to host this stuff, they'll lock your entire account and blacklist your device and SSH keys across the platform.

Don't upload the code. Don't create the iframe. Don't link to the game.
If you want to document the project, write about it in text. Explain how the engine works, the technical decisions you made, all of that. Discussing reverse engineering in a README is fine. Giving people a working way to play a pirated game is not. Your GitHub account is worth more than this. Don't risk it.
